design-system/react/src/components/Asset/asset-list.ts

```diff
@@ -0,0 +1,9 @@
+import type { AssetList } from './defs';
+
+export const assets: AssetList = {
+  eth: {
+    symbol: 'ETH',
+    name: 'Ethereum',
+    imageUrl: '/assets/eth.svg',
+  },
+};
```

Comment:
We should not include this here, but instead, we can receive a JSON with these configs on the application level. This is because we will need an asset package with only asset information and images, that will contain more information.

Reply:
✅